lib/LedgerSMB/Report/Unapproved/Batch_Detail.pm

```diff
@@ -254,10 +257,14 @@ sub run_report{
     },
     ]);
 
+    # Currently AR/AP screen is the only way to check receipt and payment
```

Comment:
what do you mean by this comment? do you mean the actual AR/AP transactions&invoices?

Reply:
Yes, the actual AR/AP Transactions & Invoices. After the payment/receipt are made from Cash >  Payment/Receipt or Cash > Vouchers > Payments/Receipts, there is no way to review that. And Cash > Reports > Payments/Receipts are not actually useful in real life. That's why I add `script` as ar and ap to check for receipt and payment respectively.